fitbenchmarking/utils/options.py

```diff
@@ -35,18 +35,18 @@ class Options(object):
          'ralfit': ['gn', 'gn_reg', 'hybrid', 'hybrid_reg'],
          'scipy': ['Nelder-Mead', 'Powell', 'CG', 'BFGS',
                    'Newton-CG', 'L-BFGS-B', 'TNC', 'SLSQP'],
-         'scipy_ls': ['lm-scipy-no-jac', 'lm-scipy', 'trf',
-                      'dogbox']}
+         'scipy_ls': ['lm-scipy', 'trf', 'dogbox']}
     VALID_FITTING = \
         {'algorithm_type': ['all', 'ls', 'deriv_free', 'general'],
          'software': ['bumps', 'dfo', 'gsl', 'levmar', 'mantid', 'minuit',
                       'ralfit', 'scipy', 'scipy_ls'],
-         'jac_method': ['scipy', 'analytic', 'numdifftools'],
+         'jac_method': ['analytic', 'scipy', 'default', 'numdifftools'],
          'cost_func_type': ['nlls', 'weighted_nlls', 'hellinger_nlls',
                             'poisson']}
     VALID_JACOBIAN = \
         {'scipy': ['2-point', '3-point', 'cs'],
          'analytic': ['cutest'],
+         'default': ['default'],
```

Comment:
This isn't consistent with the docs, where 'default' is replaced with 'solver'

![Screenshot 2021-05-10 133239](https://user-images.githubusercontent.com/79837359/117659829-55576880-b194-11eb-87b0-165bf116e237.png)